### PR TITLE
Add option to also add tasks with MM-DD, being YYYY current year

### PR DESCRIPTION
**Pull request by alexandrebarbaruiva**

### CODE REVIEW

**Comment by Thor77** [comment]
```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 03-01, _f: undefined, _strict: undefined, _locale: [object Object]
Error
[...]
wunderline-add.js:106
    dueDate = dueDate.format("YYYY-MM-DD");
                      ^

TypeError: dueDate.format is not a function
    at main (wunderline-add.js:106:23)
[...]
```
Doesn't look like it's working correctly.
Adding two dates seems to result in an integer representing the unix timestamp of the date and your "calculated" year is not considered at all.

File: wunderline-add.js [hunk 1]
```diff
@@ -92,6 +92,10 @@ function main() {
   if (app.due) {
     if (moment(app.due, "YYYY-MM-DD", true).isValid()) {
       dueDate = moment(app.due);
+    } else if (moment(app.due, "MM-DD", true).isValid()) {
+      var date = new Date();
+      var year = date.getFullYear();
+      dueDate = year + moment(app.due);
```
Comment (Thor77) [inline wunderline-add.js]: It doesn't look like this is working like you expect it to.

**Review by Thor77 — CHANGES_REQUESTED** [review]

File: wunderline-add.js (hunk 1 above)
Comment (alexandrebarbaruiva) [reply wunderline-add.js]: How do I test on my machine? I never worked with javascript before 😅

Comment (Thor77) [reply wunderline-add.js]: Clone your fork, run `npm install` and use `node wunderline.js` where you would normally use just `wunderline`.

Comment (alexandrebarbaruiva) [reply wunderline-add.js]: Thanks!

**Comment by alexandrebarbaruiva** [comment]
Fixed, should be working now

**Review by Thor77 — CHANGES_REQUESTED** [review]
Please don't include this change in this PR but open a separate one.

**Comment by alexandrebarbaruiva** [comment]
Yes, I did that by mistake, sorry. Fixed now

**Review by Thor77 — APPROVED** [review]
Looks good to me :+1:

**Comment by alexandrebarbaruiva** [comment]
Can this PR be merged or is there anything more to do?

**Review by wayneashleyberry — APPROVED** [review]

**Comment by wayneashleyberry** [comment]
@Thor77 would you like to publish a new release with all of these updates?

**Comment by Thor77** [comment]
@wayneashleyberry Yeah, will do. Should it be a major one because of the changes to the general logic of the application?

**Comment by wayneashleyberry** [comment]
Technically no, this release doesn’t break backwards compatibility. ie. You should be able to run the same commands and get the same output. I’d say it’s a minor release because it adds new features.

**Comment by Thor77** [comment]
@wayneashleyberry alright, done :)

**Comment by wayneashleyberry** [comment]
Amazing, thank you so much @Thor77

**Pull request merged**